src/test/librbd/fsx.cc

```diff
@@ -2504,8 +2504,10 @@ __aio_rw(int rw, int fd, char *buf, unsigned len, unsigned offset)
 	} while (ret == -EINTR);
 
 	if (ret != 1) {
-		if (ret == 0)
+		if (ret == 0) {
```

Comment:
please note that this is dead code in ceph. it's never executed in the ceph variant of fsx. instead, we use `{p,librbd_}{write,read}` for kbd/rbd-nbd and rbd modes. i wonder if we can just remove the aio backend for the read/write tests.